### ISSUE TITLE
No compression?

### ISSUE THREAD

**Issue by lab313ru** [joesblog/PowerPackerWin#1]
So, is it only for unpacking, no compressor?

**Comment by joesblog** [comment]
That's correct, I wrote it purely to extract old txt(PP) files from places like aminet, I could probably expand it to do the compression. or better yet rewrite the older C code and turn it into C#, I just don't have the time.

**Comment by lab313ru** [comment]
Here is the compressor and decompressor: https://github.com/lab313ru/powerpacker_src
